libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -19,6 +19,18 @@
         <mat-icon>remove_circle</mat-icon>
       </button>
     </div>
+    <div class="finished-text" *ngIf="readingItem.finished">
+      Finished on: {{ readingItem.finishedDate | date:'MM/dd/yyyy' }}
```

Comment:
I like that you've formatted the date but remember that only the US uses this format. Ideally you would either figure out the users's region or just use ISO8601

Reply:
Addressed.
<img width="1613" alt="Screen Shot 2020-10-01 at 9 37 12 PM" src="https://user-images.githubusercontent.com/19898265/94883204-a8fc1980-042f-11eb-84f4-0c3c0b7cb17f.png">